index.js

```diff
@@ -17,34 +19,34 @@ program
   .parse(process.argv);
 
 // judge options
-let options=[];
+const options = ['--patience'];
```

Comment:
Adding `--patience` makes the last hunk of this file much easier to read.

![image](https://user-images.githubusercontent.com/73085/29154896-d741213c-7dd2-11e7-96c3-0bf884a7a92b.png)